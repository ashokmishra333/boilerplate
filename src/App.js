import React from "react";
import CheckboxWithLabel from "./CheckboxWithLabel";

function App() {
  return (
    <h1 className="container">
      Hello World! <CheckboxWithLabel />
    </h1>
  );
}

export default App;
