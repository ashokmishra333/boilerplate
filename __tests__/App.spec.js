import React from "react";
import renderer from "react-test-renderer";
import App from "../src/App";

describe("App comonent", () => {
  it("Snapshot Testing", () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
