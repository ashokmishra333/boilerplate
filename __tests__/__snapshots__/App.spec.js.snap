// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App comonent Snapshot Testing 1`] = `
<h1
  className="container"
>
  Hello World!
</h1>
`;
